mp/src/materialsystem/stdshaders/pbr_ps30.fxc

```diff
@@ -321,7 +345,9 @@ float4 main( PS_INPUT i ) : COLOR
 
 	float fogFactor = CalcPixelFogFactor( PIXELFOGTYPE, g_FogParams, g_EyePos.z, i.worldPos.z, i.projPos.z );
 
-	float alpha = 1.0;
+	//float alpha = 1.0;
+	float alpha = albedo.a; // Gocnak told me to add this
```

Comment:
This is ok, can you remove the comment above and to the right?

Reply:
done